static/js/apps/nl_eval/setence_section.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from "react";
+
+import { SearchResult } from "./search_result";
+import { BASE_URL, EmbeddingObject } from "./util";
+
+export interface SentenceSectionProps {
+  sentence: string;
+  modelNames: string[];
+  goldenStatVars: string[];
+  overrideStatVars: Record<string, EmbeddingObject[]>;
+}
+
+export function SentenceSection(props: SentenceSectionProps): JSX.Element {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const toggleTableVisibility = () => {
+    setIsExpanded(!isExpanded);
+  };
+  return (
+    <div className="setence-container">
```

Comment:
nit: sentence-container

also rename file to "sentence_section"?

Reply:
Done